dashboard/config/application.rb

```diff
@@ -22,6 +23,8 @@ class Application < Rails::Application
 
     config.encoding = 'utf-8'
 
+    Rails.application.routes.default_url_options[:host] = CDO.canonical_hostname('studio.code.org')
```

Comment:
Yes, the link_to method raised an exception that this needed to be defined for link_to to work when I changed the method from `:sendmail` to my class. I found this solution here: http://stackoverflow.com/questions/22288951/actionviewtemplateerror-missing-host-to-link-to-please-provide-the-host-p